### ISSUE TITLE
Hotbar stops working

### ISSUE THREAD

**Issue by gabbigamers** [ControlCompany/InfectedCompany#21]
So when i got infected and i transformed and when i transformed back no buttons or anything worked for me anymore. I became useless.

**Comment by 588-th** [comment]
Well, I have faced a similar issue. The only difference is, it could happen under any circumstances. I accessed the console and saw the following error message ([image1](https://github.com/ControlCompany/InfectedCompany/assets/101037952/3e478d21-d4f8-4c1d-8fc8-8a911753f149)).

It helped me to create a new profile in r2modman without any other mods. I guess the problem was compatibility with some kind of mod. Moreover, removing this certain mod did not help, only creating a new clean profile.

Since I couldn't fully figure out what the problem was and which mod might be compatible with, if that's the problem, I didn't create an issue. But perhaps this is the same problem that you are facing.

**Comment by gabbigamers** [comment]
Yeah i suppose it is the same problem. Wish this mod was more compatible with other mods lol. I only had other monster mods like the fiend or the locker and more.
